frontend/.env

```diff
@@ -0,0 +1 @@
+WDS_SOCKET_PORT=3001
```

Comment:
why are we checking in .env files, perhaps there should be a .env-sample we could check in.  Also standard practices are not to check in .env files.  How do you see this being used in openshift?